### PR TITLE
[v5.0] consistent naming of keys

### PR DESCRIPTION
**Pull request by gggina**

### CODE REVIEW

File: stackpacks/integrations/openshift.md [hunk 1]
```diff
@@ -111,10 +111,10 @@ To remedy this situation, the kubernetes\_state check can be configured to run a
   helm upgrade --install \
   --namespace stackstate \
   --create-namespace \
-  --set-string 'stackstate.apiKey'='<your-api-key>' \
+  --set-string 'stackstate.apiKey'='<STACKSTATE_RECEIVER_API_KEY>' \
   --set-string 'stackstate.cluster.name'='<your-cluster-name>' \
```
Comment (julianosk) [inline stackpacks/integrations/openshift.md:115]: Maybe here you put `OPENSHIFT_CLUSTER_NAME` as in the stackpack.

File: stackpacks/integrations/aws/aws.md [hunk 2]
```diff
@@ -213,10 +213,10 @@ If StackState Agent is running on Kubernetes, the AWS check should be configured
 helm upgrade --install \
 --namespace stackstate \
 --create-namespace \
---set-string 'stackstate.apiKey'='<your-api-key>' \
+--set-string 'stackstate.apiKey'='<STACKSTATE_RECEIVER_API_KEY>' \
 --set-string 'stackstate.cluster.name'='<your-cluster-name>' \
```
Comment (julianosk) [inline stackpacks/integrations/aws/aws.md:217]: Maybe `<KUBERNETES_CLUSTER_NAME>` here

File: stackpacks/integrations/aws/aws.md [hunk 3]
```diff
@@ -227,10 +227,10 @@ stackstate-cluster-agent stackstate/cluster-agent
...
```
Comment (julianosk) [inline stackpacks/integrations/aws/aws.md:231]: Maybe <KUBERNETES_CLUSTER_NAME> here

File: stackpacks/integrations/azure.md [hunk 4]
```diff
@@ -67,15 +67,15 @@ To enable the Azure integration and begin collecting data from Azure, you will n
...
     <Azure tenantId> \
-    {{config.baseUrl}} \
-    {{config.apiKey}} \
-    <Azure subscriptionId> \
-    <Azure clientId> \
-    <Azure clientSecret> \
-    <Azure resourceGroupName>
+    <STACKSTATE_BASE_URL> \
+    <STACKSTATE_RECEIVER_API_KEY> \
+    <AZURE_subscriptionId> \
```
Comment (julianosk) [inline stackpacks/integrations/azure.md:72]: Should we have all keys uppercase? (to be consistent)

File: stackpacks/integrations/azure.md [hunk 5]
```diff
@@ -67,15 +67,15 @@ To enable the Azure integration and begin collecting data from Azure, you will n
...
+    <AZURE_clientId> \
+    <AZURE_clientSecret> \
+    <AZURE_resourceGroupName>
 ```
 
-You can also specify two additional (positional) arguments after the `<Azure resourceGroupName>`: 
+You can also specify two additional (positional) arguments after the `<AZURE_resourceGroupName>`:
```
Comment (julianosk) [inline stackpacks/integrations/azure.md:78]: Should we have all keys uppercase? (to be consistent)
```suggestion
You can also specify two additional (positional) arguments after the `<AZURE_RESOURCE_GROUP_NAME>`: 
```

File: stackpacks/integrations/azure.md [hunk 6]
```diff
@@ -87,12 +87,12 @@ az login
...
--subscriptionId <azure subscriptionId> `
--servicePrincipalId <Client Id> `
--servicePrincipalSecret <Client Secret> `
--resourceGroupName <Resource GroupName to deploy to>
+-stsApiUrl <STACKSTATE_BASE_URL> `
+-stsApiKey <STACKSTATE_RECEIVER_API_KEY> `
+-subscriptionId <AZURE_subscriptionId> `
+-servicePrincipalId <AZURE_ClientId> `
+-servicePrincipalSecret <AZURE_ClientSecret> `
+-resourceGroupName <RESOURCE_GroupName_TO_DEPLOY_TO>
```
Comment (julianosk) [inline stackpacks/integrations/azure.md:95]:
```suggestion
-resourceGroupName <AZURE_RESOURCE_GROUP_NAME>
```

File: stackpacks/integrations/kubernetes.md [hunk 7]
```diff
@@ -111,10 +111,10 @@ To remedy this situation, the kubernetes\_state check can be configured to run a
   helm upgrade --install \
   --namespace stackstate \
   --create-namespace \
-  --set-string 'stackstate.apiKey'='<your-api-key>' \
+  --set-string 'stackstate.apiKey'='<STACKSTATE_RECEIVER_API_KEY>' \
   --set-string 'stackstate.cluster.name'='<your-cluster-name>' \
```
Comment (julianosk) [inline stackpacks/integrations/kubernetes.md:115]:
```suggestion
  --set-string 'stackstate.cluster.name'='<KUBERNETES_CLUSTER_NAME>' \
```

**Review by julianosk — COMMENTED** [review]
It looks great! Only a few things to make the docs even more consistent.

File: stackpacks/integrations/openshift.md (hunk 1 above)
Comment (gggina) [reply stackpacks/integrations/openshift.md:115]: done

File: stackpacks/integrations/aws/aws.md (hunk 2 above)
Comment (gggina) [reply stackpacks/integrations/aws/aws.md:217]: done

File: stackpacks/integrations/aws/aws.md (hunk 3 above)
Comment (gggina) [reply stackpacks/integrations/aws/aws.md:231]: done

File: stackpacks/integrations/azure.md (hunk 5 above)
Comment (gggina) [reply stackpacks/integrations/azure.md:78]: I'm going to leave all of the Azure references in the format `<AZURE_keyName>` they come up too often throughout the azure page and this is the most consistent approach

File: stackpacks/integrations/kubernetes.md (hunk 7 above)
Comment (gggina) [reply stackpacks/integrations/kubernetes.md:115]: done

File: stackpacks/integrations/azure.md (hunk 4 above)
Comment (gggina) [reply stackpacks/integrations/azure.md:72]: see below `<AZURE_resourceGroupName>`

**Pull request merged**